.github/workflows/sync-release-branch-2023b.yml

```
@@ -0,0 +1,35 @@
+# This GitHub action is meant to be triggered weekly in order to sync the upstream release branch with the downstream branch fork
+
+name: Sync downstream release branch with upstream's
```

Would it be possible to include the release branch here so we can distinguish between these two? E.g.:

```suggestion
name: Sync downstream release-2023b branch with upstream's
```

Would be nice to update the very same way also the existing action yaml file for the 2023a